### PR TITLE
Update Geocoding Beta materials

### PR DESCRIPTION
**Pull request by millzpaugh**
This provides several updates for the geocoding pilot (now geocoding beta program): 
- Changes  the name of the geocoding "pilot" to geocoding Beta
- Adds ticket title template 
- Has calendly open in a new tab
- Takes Pilot out of "Becoming partner ticket", Calendly event, and overview doc
- Auto assigns issues to Nagela & Ann
- Adds language to explicitly state that users create ticket and THEN schedule via calendly

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 
-name: Become a Geocoding Pilot Partner
+name: Become a Geocoding Beta Program
```
Comment (spencer-skylight) [inline .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:3]: The wording here seems off.  Maybe `Register for the Geocoding Beta Program`?

File: .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-title: ''
-labels: 'geocoding-pilot'
-assignees: '@millzpaugh,@NagelaNukuna'
+title: [Affiliated Jurisdiction] Geocoding Beta Program
+labels: 'geocoding-beta'
+assignees: @millzpaugh,@NagelaNukuna
 
 ---
-Thank you for your interest in PHDI’s Geocoding Pilot Partnership! See full details about the pilot [here](https://github.com/CDCgov/phdi/blob/main/docs/geocoding_pilot/overview.md). Please fill in the information below and we will be in touch shortly with next steps.
+Thank you for your interest in PHDI’s Geocoding Beta Partnership! See full details about the beta [here](https://github.com/CDCgov/phdi/blob/main/docs/geocoding_pilot/overview.md){:target="_blank"}. Please fill in the information below and we will be in touch shortly with next steps.
```
Comment (spencer-skylight) [inline .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:10]: Do you want to suggest that they replace the [Affiliated Jurisdiction] placeholder with their jurisdiction name?

File: docs/geocoding_pilot/overview.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Public Health Data Infrastructure (PHDI) Geocoding Pilot Partnership 2022 
+# Public Health Data Infrastructure (PHDI) Geocoding Beta Program 2022
```
Comment (spencer-skylight) [inline docs/geocoding_pilot/overview.md:1]: Do you want to rename the folder with "beta"?

File: .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md (hunk 1 above)
Comment (spencer-skylight) [inline .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:3]: Do you want to rename the template with "beta"?

**Review by spencer-skylight — APPROVED** [review]
I left a few comments I think you'll want to address, but am approving so you can move forward quickly when you are ready. Thanks!

File: .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md (hunk 1 above)
Comment (millzpaugh) [reply .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:3]: Yup. Thank you!

Comment (millzpaugh) [reply .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:3]: Yes!

File: .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (millzpaugh) [reply .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:10]: Opening the link in a new tab.

Comment (millzpaugh) [reply .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:10]: Great suggestion.

File: .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md [hunk 4]
```diff
@@ -17,9 +17,12 @@ Please provide your:
...
 - Role
 - Affiliated Public Health Department (link url if possible)
-## Schedule a time to receive your subsidized Smarty API key
 
-Schedule a meeting here via [Calendly](https://calendly.com/ann-299/geocoding-pilot-partner).
+Now, **create your ticket!** Once the ticket has been created, schedule time to connect with a PHDI team member using the Calendly link listed below. 
+
+## Schedule a time to connect with PHDI and receive your subsidized Smarty API key
+
+Schedule a meeting here via [Calendly](https://calendly.com/ann-299/geocoding-pilot-partner){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (millzpaugh) [reply .github/ISSUE_TEMPLATE/geocoding_pilot_partner.md:25]: This is syntax for opening in new tab

File: docs/geocoding_pilot/overview.md (hunk 3 above)
Comment (millzpaugh) [reply docs/geocoding_pilot/overview.md:1]: Yes

**Pull request merged**